core/Files.cc

```diff
@@ -346,11 +346,9 @@ bool File::permitOverloadDefinitions() const {
 }
 
 bool File::isPackage() const {
-    return flags.isPackage;
-}
-
-void File::setIsPackage(bool isPackage) {
-    this->flags.isPackage = isPackage;
+    // If the `__package.rb` file is at `typed: ignore`, then we haven't even parsed it.
+    // Any loop over "all package files" is really only wants "all non-ignored package files."
```

Comment:
```suggestion
    // Any loop over "all package files" really only wants "all non-ignored package files."
```

Reply:
Thanks, I will be sure to include this in the local branch I have stacked on this change.